TestCases/compliance-level-3/0070-feel-instance-of/0070-feel-instance-of.dmn

```diff
@@ -173,19 +227,27 @@
         </literalExpression>
     </decision>
 
-<!--     <decision name="number_011" id="_number_011">
+<!--    <decision name="number_011" id="_number_011">
         <variable name="number_011"/>
         <literalExpression>
-            <text>123.01 instance of context</text>
+            <text>123.01 instance of context&lt;&gt;</text>
         </literalExpression>
-    </decision> -->
+    </decision>-->
 
-<!--     <decision name="number_012" id="_number_012">
+    <decision name="number_012" id="_number_012">
         <variable name="number_012"/>
         <literalExpression>
-            <text>123.01 instance of function</text>
+            <text>123.01 instance of function&lt;&gt;-&gt;Any</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="number_013" id="_number_013">
+        <variable name="number_013"/>
+        <literalExpression>
+            <!-- true: does not take allowedValues into account -->
+            <text>256 instance of t255</text>
```

Comment:
We are revising this PR/tests, and while I fully agree with this test, I am wondering how to best report this back to the RTF.

On one side, `allowedValues` is a concept of the DMN metamodel.
On the other side, DMN specification at a point introduced for `instance of` to be working with the types defined in the DMN model, albeit there is no way to do equivalently in pure FEEL.

As an end-result, this case is uncovered which might be a bit counter-intuitive...